tests/cases.py

```diff
@@ -966,7 +966,7 @@ def test_cli_training_nations_testing(self):
 
     def _help_test_cli(self, args):
         """Test running the pipeline on all models."""
-        if issubclass(self.cls, pykeen.models.RGCN):
+        if issubclass(self.cls, (pykeen.models.RGCN, pykeen.models.ERModel)):
```

Comment:
this isn't going to work the way you expect since many models are a subclass of pykeen.models.ERModel